News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsReaderListFragment.java

```diff
@@ -355,6 +358,12 @@ public void bindUserInfoToUI() {
             UserInfo userInfo = (UserInfo) fromString(uInfo);
             if (userInfo.displayName != null)
                 userTextView.setText(userInfo.displayName);
+                Glide
```

Comment:
Since we're already using the Universal-Image-Loader I would prefer not to include another image loading library such as glide. The following code should pretty much do the same by using the universal-image-loader library

```java
int placeHolder = R.mipmap.ic_launcher;
DisplayImageOptions displayImageOptions = new DisplayImageOptions.Builder()
                //.displayer(new RoundedBitmapDisplayer((int) 20))
                .displayer(new CircleBitmapDisplayer()) // you might need to use an ImageViewAware..? Instead of the normal ImageView. Not 100% sure though
                .showImageOnLoading(placeHolder)
                .showImageForEmptyUri(placeHolder)
                .showImageOnFail(placeHolder)
                .cacheOnDisk(true)
                .cacheInMemory(true)
                .build();

String avatarUrl = mOc_root_path + "/index.php/avatar/" + Uri.encode(userInfo.userId) + "/64";
ImageLoader.getInstance().displayImage(avatarUrl, this.headerLogo, displayImageOptions);
```

Reply:
No problem, i will update the PR (might take a few days)

Reply:
Done.